src/transformStateWithClones.js

```diff
@@ -6,8 +6,33 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const saveState = [];
+  const currentState = { ...state };
```

Comment:
```suggestion
  let currentState = { ...state };
``` just for next suggestion

Reply:
just for next suggestion